### PR TITLE
Mixih/release0.9.1

### PR DESCRIPTION
**Pull request by Mixih**
### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by Mixih** [comment]
CC @sils as requested

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+
+This bugfix release addressed the following issues:
+
+- The installer has been fixed to only install on supported python versions.
+  `Issue #3310 <https://github.com/coala/coala/issues/3310>`_
+  `Issue #3383 <https://github.com/coala/coala/issues/3383>`_
+- The format_str option to specify a format string has ben restored to
+  coala run in format mode. This is deprecated and was only restored to
+  provide plug-in developers a grace period to switch to the replacement
+  setting "format"
```
Comment (sils) [inline RELEASE_NOTES.rst]: missing period, also use backticks for `format`

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+  `Issue #3303 <https://github.com/coala/coala/issues/3303>`_
+- Setuptools was removed from our requirements list because having it only
+  only caused dependency problems for other packages installed on the system
+  `Coala Bears Issue #751 <https://github.com/coala/coala-bears/issues/751>`_
```
Comment (sils) [inline RELEASE_NOTES.rst]: coala is always written with a lower case c

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3412/commits/5cd1e71f5b4a88eb7bb7e32b87751eb6c1a47d11

**Comment by sils** [comment]
unack https://github.com/coala/coala/pull/3412/commits/a634e1e3b9bf5aef4c3292a85457385ec12494b7

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3412/commits/a60b5767b7faea0346e3c0258230d9ea5b2d2694 https://github.com/coala/coala/pull/3412/commits/9ed38627f21e939fc17b54e9863a92972ec81f37

**Comment by sils** [comment]
@rultor merge

**Pull request merged**